### ISSUE TITLE
Cross validation

### ISSUE THREAD

**Issue by aisha-lawal** [aisha-lawal/inrmorph#13]
We do cross validation (using 3 subjects to train 2 subjects to test) to select optimal values of parameters, basic parameters will be learning rate and regularisation term weights.

- [x] select which regularisation hyperparameters are needed to for cross val
- [x] write cross val script
